nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -245,6 +245,66 @@
      options.
    </para>
   </listitem>
+  <listitem>
+   <para>
+     The <literal>nscd</literal> now disables all caching of
+     <literal>passwd</literal> and <literal>group</literal> databases by
+     default. This was interferring with the correct functioning of the
+     <literal>libnss_systemd.so</literal> module which is used by
+     <literal>systemd</literal> to manage uids and usernames in the presence
+     of <literal>DynamicUser=</literal> in systemd services.
+     The was already the default behaviour in presence of
+     <literal>services.sssd.enable = true</literal> because nscd caching
+     would interfere sssd in unpredictable ways as well.Because we're using nscd
```

Comment:
```suggestion
     would interfere sssd in unpredictable ways as well. Because we're using nscd
```
(from https://github.com/NixOS/nixpkgs/pull/50316#discussion_r235370732)